src/addons/Portal/Portal.js

```diff
@@ -344,14 +348,15 @@ class Portal extends Component {
     if (!this.state.open) return
     debug('renderPortal()')
 
-    const { children, className, eventPool } = this.props
+    const { children, className, eventPool, style } = this.props
 
     this.mountPortal()
 
     // Server side rendering
     if (!isBrowser()) return null
 
     this.rootNode.className = className || ''
+    this.rootNode.style = style ? toInlineStyle(style) : ''
```

Comment:
Nice catch!

Reply:
We also no longer need this external library based on the refactor requested below.